### PR TITLE
Endpoints setters

### PR DESCRIPTION
**Pull request by thiagocordeiro**
In order to make easier to switch between development, staging and production environment this changes provide a way to setup de URLs on the project which requires it, this values are not mandatory and production values are set as default

### CODE REVIEW

**Review by Smitsel — APPROVED** [review]

File: src/Provider/Mollie.php [hunk 1]
```diff
@@ -72,6 +72,42 @@ public function __construct(array $options = [], array $collaborators = [])
...
 
+	/**
+	 * @var string
+	 */
+	private $mollieApiUrl = self::MOLLIE_API_URL;
+
+	/**
+	 * @var string
+	 */
+	private $mollieWebUrl = self::MOLLIE_WEB_URL;
```
Comment (ShortlyMAB) [inline src/Provider/Mollie.php]: can we define these properties before `__construct` method, please?

File: src/Provider/Mollie.php [hunk 2]
```diff
@@ -72,6 +72,42 @@ public function __construct(array $options = [], array $collaborators = [])
...
+
+	/**
+	 * Define Mollie api URL
+	 *
+	 * @param string $url
+	 * @return Mollie
+	 */
+	public function setMollieApiUrl ($url)
```
Comment (ShortlyMAB) [inline src/Provider/Mollie.php]: **NBC**: Setting this value in the constructor with checking the `$options` parameter makes more sense to me. Same for `setMollieWebUrl()`.

Comment (thiagocordeiro) [reply src/Provider/Mollie.php]: Completely :)

**Review by ShortlyMAB — APPROVED** [review]

**Comment by willemstuursma** [comment]
In the generic PHP API client it is done with setters only, so not an option.

**Comment by willemstuursma** [comment]
Regardless, the change is fine with me but I would prefer setters.

**Comment by Smitsel** [comment]
So before i approved it was setters, and i kind of prefer it that way myself too.
It's also a consistency thing across our SDK's as the mollie-api-php also does it like that.

What do you think @actuallymab ?

**Comment by ShortlyMAB** [comment]
I wanted it to be in construct because changing this value after the initialization of the class did not mean something to me. In this case, I understand your concerns about consistency, since @thiagocordeiro is on holiday, I will commit that change so that we can merge this.

**Comment by Smitsel** [comment]
Cool! Much appreciated!

**Pull request merged**